jsonargparse_tests/test_alias.py

```diff
@@ -0,0 +1,10 @@
+def test_alias():
```

Comment:
Note that this repo uses the standard unittest. Simply adding a function that starts with `test_` will not run as part of the tests.